src/tests/ftest/pool/info_tests.py

```diff
@@ -21,128 +21,122 @@
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
 """
-import os
-import json
-from apricot import Test
+# import os
```

Comment:
If not needed these commented lines can be removed.

Reply:
Thanks I missed these.